apps/modernization-ui/src/apps/patient/add/extended/AddPatientExtended.tsx

```diff
@@ -32,6 +40,28 @@ export const AddPatientExtended = () => {
 
     const handleSave = form.handleSubmit(interaction.create);
 
+    const handleCancel = () => {
+        if (value) {
+            closeForm();
+        } else {
+            setCancelModal(true);
+        }
+    };
+
+    const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+        setVisibilityCheckBox(e.target.checked);
+    };
+
+    const handleConfirmCancel = () => {
+        save(visibilityCheckBox);
+        closeForm();
+    };
+
+    const closeForm = () => {
+        form.reset();
+        history.go(-1);
+    };
```

Comment:
Question: should clicking cancel be the same as pushing the back button or should it have defined behavior (go to a certain route)

Reply:
For my understanding, you can only go to extended form through the basic form, and the expected behavior is to go back to the basic form. Should I be more intentional with the redirect?

Reply:
You could go directly to the URL, is the `form.reset` requried?